file-formats/chko/chko-v1.json

```diff
@@ -108,7 +110,7 @@
             "maxLength": 255
           },
           "hidden": {
-            "title": "Is Hidden",
+            "title": "Is hidden",
```

Comment:
Is this an intended change to the title?

Reply:
No, it's not intended. Thanks. I'll fix it.

Reply:
Fixed